### ISSUE TITLE
Player tilt bug

### ISSUE THREAD

**Issue by rube200** [SmartlyDressedGames/Unturned-3.x-Community#2507]
Q always has priority over E and others...
https://streamable.com/z0us72

**Comment by SDGNelson** [comment]
Well they are opposing inputs, but I suppose the game should treat Q+E as 0.

**Comment by rube200** [comment]
I agree with that and please also fix the instant lift when the player is lying down
